sklearn/datasets/base.py

```diff
@@ -622,21 +633,29 @@ def load_linnerud(return_X_y=False):
         'targets', the two multivariate datasets, with 'data' corresponding to
         the exercise and 'targets' corresponding to the physiological
         measurements, as well as 'feature_names' and 'target_names'.
+        In addition, you will also have access to 'data_filename'
+        (added in version 0.20), the physical location of linnerud data csv
+        dataset, and 'target_filename' (added in version 0.20), the
+        physical location of linnerud targets csv datataset.
```

Comment:
I am not in favor of  `(added in version 0.20)` in the narrative.

I would add a directive on the bottom of data

```python
        the physical location of iris csv dataset.

        .. versionadded:: 0.20
           The ``filename`` attribute.
```